bundles/core/src/test/resources/graphql/magento-graphql-productcarousel.json

```diff
@@ -0,0 +1,122 @@
+{
+    "products": {
+      "items": [
+        {
+          "__typename": "SimpleProduct",
+          "id": 1073,
+          "sku": "170227049",
+          "name": "Hiking Poles",
+          "thumbnail": {
+            "label": "Hiking Poles",
+            "url": "https://staging-5em2ouy-7ztex4hq2b6mu.us-3.magentosite.cloud/media/catalog/product/cache/74c1057f7991b4edb2bc7bdaa94de933/1/7/170227049_2.jpg"
+          },
```

Comment:
Hostname in url could be a dummy url than actual for test.

Reply:
Url replaced with dummy url